Annotation processing exception: "failed to analyze: java.lang.reflect.InvocationTargetException"

Lib "actions-dispatcher": 3.0.2
Kotlin: 1.5.21

Build error:
```
java.lang.IllegalStateException: failed to analyze: java.lang.reflect.InvocationTargetException
	at org.jetbrains.kotlin.analyzer.AnalysisResult.throwIfError(AnalysisResult.kt:56)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler.compileModules$cli(KotlinToJVMBytecodeCompiler.kt:193)
...
Caused by: java.lang.IllegalStateException: Could not parse metadata! This should only happen if you're using Kotlin <1.1.
	at com.squareup.kotlinpoet.metadata.KotlinPoetMetadata.readKotlinClassMetadata(KotlinPoetMetadata.kt:83)
	at com.squareup.kotlinpoet.metadata.KotlinPoetMetadata.toImmutableKmClass(KotlinPoetMetadata.kt:113)
	at com.squareup.kotlinpoet.metadata.KotlinPoetMetadata.toImmutableKmClass(KotlinPoetMetadata.kt:46)
	at com.github.rougsig.actionsdispatcher.kapt.ActionDispatcherProcessor$process$$inlined$forEach$lambda$1.invoke(ActionDispatcherProcessor.kt:77)
```

Same error https://discuss.kotlinlang.org/t/annotation-processing-exception-failed-to-analyze-java-lang-reflect-invocationtargetexception/15518

Solution https://stackoverflow.com/questions/7687829/java-6-annotation-processing-getting-a-class-from-an-annotation/7688029#7688029

--- a/buildSrc/src/main/kotlin/Deps.kt
+++ b/buildSrc/src/main/kotlin/Deps.kt
@@ -1,11 +1,11 @@
 object Deps {
-  private const val KOTLIN_VERSION = "1.4.0"
+  private const val KOTLIN_VERSION = "1.5.21"
   const val KOTLIN_GRADLE = "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
 
   const val KOTLIN_STD = "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
   const val KOTLIN_REFLECT = "org.jetbrains.kotlin:kotlin-reflect:$KOTLIN_VERSION"
 
-  private const val KOTLINPOET_VERSION = "1.6.0"
+  private const val KOTLINPOET_VERSION = "1.9.0"
   const val KOTLINPOET = "com.squareup:kotlinpoet:$KOTLINPOET_VERSION"
   const val KOTLINPOET_METADATA = "com.squareup:kotlinpoet-metadata:$KOTLINPOET_VERSION"
 

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,4 +1,4 @@
-VERSION_NAME=3.0.2
+VERSION_NAME=3.0.3
 VERSION_CODE=1
 GROUP=com.github.rougsig
 POM_DESCRIPTION=A simple annotation processor for generating actions dispatcher

